<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_select_dialog</class>
 <widget class="QMainWindow" name="user_select_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Steam account</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QPushButton" name="close_button">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="info_label">
        <property name="text">
         <string>Enter a Steam account ID or select a user</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="steam_url_text">
        <property name="text">
         <string>76561198055087665</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="select_account_button">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="player_dropdown"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>user_select_dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
